presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -96,7 +89,10 @@ Password authentication needs to be configured to use LDAP. Create an
 Property                                                Description
 ======================================================= ======================================================
 ``ldap.url``                                            The url to the LDAP server. The url scheme must be
-                                                        ``ldaps://`` since Presto allows only Secure LDAP.
+                                                        ``ldap://`` or ``ldaps://``. Connecting to the LDAP
+                                                        server without SSL enabled requires
+                                                        ``ldap.allow-insecure=true``.
+``ldap.allow-insecure``                                 Allow insecure connection to the LDAP server.
```

Comment:
This property should be renamed... insecure does NOT mean the same as "not secured" .. https://www.merriam-webster.com/dictionary/insecure. At least the description should change.

Reply:
Maybe something like "Allow using an LDAP connection that is not secured with TLS."

Reply:
I'm not an expert in this area but it's pretty popular naming convention, eg. chrome has `allow-insecure-localhost`.

Reply:
popular doesnt mean its correct. I would try to avoid since its plain wrong.

Reply:
@electrum @kokosing wdyt?